src/app/src/main/kotlin/org/vaccineimpact/api/app/controllers/ModellingGroupController.kt

```diff
@@ -116,27 +116,6 @@ open class ModellingGroupController(context: ControllerContext)
         return data
     }
 
-    fun postBurdenEstimates(context: ActionContext, estimateRepository: BurdenEstimateRepository): String
-    {
-        // First check if we're allowed to see this touchstone
-        val path = getValidResponsibilityPath(context, estimateRepository)
-
-        val request = context.request
-        if (request.raw().getAttribute("org.eclipse.jetty.multipartConfig") == null)
-        {
-            val multipartConfigElement = MultipartConfigElement(System.getProperty("java.io.tmpdir"))
-            request.raw().setAttribute("org.eclipse.jetty.multipartConfig", multipartConfigElement)
-        }
-
-        request.raw().getPart("file").inputStream.bufferedReader().use {
-
-            // Then add the burden estimates
-            val data = DataTableDeserializer.deserialize(it.readText(), BurdenEstimate::class, Serializer.instance).toList()
-            return uploadBurdenEstimates(data, estimateRepository, context, path)
-        }
-
-    }
-
```

Comment:
I didn't mean to include these changes here - sorry. However, all I have done is renamed the three burden estimate functions (as we discussed, they had confusing names) and moved them to all be next to each other.